modules/social_features/social_core/config/install/gin.settings.yml

```diff
@@ -0,0 +1,4 @@
+favicon:
```

Comment:
Hi @agami4 @navneet0693 
gin.settings.yml is something that is provided by the gin theme already

so we will be getting a
```
Drupal\Core\Config\PreExistingConfigException: Configuration objects     [error]
(gin.settings) provided by gin already exist in active configuration
in
```